docs/source/user/api.rst

```diff
@@ -476,7 +476,7 @@ Batch user creation
 
 .. code-block:: text
 
-    /api/v1/<organization-slug>/radiusbatch/
```

Comment:
As discussed in the [BatchView permissions PR](https://github.com/openwisp/openwisp-radius/pull/131#discussion_r457825262) we don't want slug in the URL and hence, I am getting rid of it.

P.S after spending some time on DRF docs & stackoverflow, I realized it's best practice to do this validation in the serializer itself, so this check is done in `RadiusOrganizationField`. :smile:

Reply:
nice thanks